staging/src/k8s.io/client-go/discovery/cached/memory/memcache_test.go

```diff
@@ -608,15 +608,15 @@ func TestMemCacheGroupsAndMaybeResources(t *testing.T) {
 		assert.False(t, memClient.receivedAggregatedDiscovery)
 		assert.True(t, memClient.Fresh())
 		// Test the expected groups are returned for the aggregated format.
-		expectedGroupNames := sets.NewString(test.expectedGroupNames...)
-		actualGroupNames := sets.NewString(groupNamesFromList(apiGroupList)...)
+		expectedGroupNames := sets.New[string](test.expectedGroupNames...)
+		actualGroupNames := sets.New[string](groupNamesFromList(apiGroupList)...)
 		assert.True(t, expectedGroupNames.Equal(actualGroupNames),
-			"%s: Expected groups (%s), got (%s)", test.name, expectedGroupNames.List(), actualGroupNames.List())
+			"%s: Expected groups (%s), got (%s)", test.name, expectedGroupNames.UnsortedList(), actualGroupNames.UnsortedList())
```

Comment:
This failure will be easier to diagnose when these slices are sorted.

For example, I find
`asdf: Expected groups (A, D, G, C, M), got (C, A, M, D)` harder to compare mentally than
`asdf: Expected groups (A, C, D, G, M), got (A, C, D, M)`

Reply:
That makes sense, thanks!